### ISSUE TITLE
[bug] Current Date not highlighted in Month View

### ISSUE THREAD

**Issue by ayushanand18** [schedule-x/schedule-x#174]
**Context:**

- Browser [e.g. chrome, safari] Google Chrome (distro: Windows 11)
- Browser-version 120.0.6099.225 (Official Build) (64-bit) (cohort: Stable) 
- Schedule-X version [e.g. v1.1.0] Please make sure that you've tried the latest version before reporting the bug.
```json
    "@schedule-x/calendar": "^1.9.0",
    "@schedule-x/event-modal": "^1.9.0",
    "@schedule-x/react": "^1.6.1",
    "@schedule-x/theme-default": "^1.9.0",
```
- NextJS 14.0.3

**Describe the bug**

The current date isn't highlighted when viewed in month view. 

**To Reproduce**

1. Embed the component in a NextJS (client rendered) component
2. Select Month View
3. Notice that the current date is not highlighted.

Sample code is as below:
```tsx
const calendar = useNextCalendarApp({
    defaultView: viewMonthGrid.name,
    views: [viewDay, viewWeek, viewMonthGrid],
    plugins: [createEventModalPlugin()],
    events: [],
    isDark: true,
  });

// ... some more code unrelated ... 

// the actual rendered Tsx
<div className="flex flex-col w-full rounded-lg shadow-md border border-slate-800 relative">
  <ScheduleXCalendar calendarApp={calendar} />
</div>
```
**Expected behavior**

The current date should be highlighted on the calendar (in month view).

**Screenshots**

![image](https://github.com/schedule-x/schedule-x/assets/36472216/35580165-29d6-4940-a06d-ec91f4a8af3c)

PS. Thanks for the amazing library!

**Comment by tomosterlund** [comment]
Thanks for reporting this! This is something I missed out on implementing, but firmly believed I already had 😁

**Comment by tomosterlund** [comment]
Released as `1.9.1`. The React component will release the update automatically later today.

**Comment by ayushanand18** [comment]
Thanks for the fix and release! That's as fast as light :)
